### PR TITLE
adding brackets to fulfil coding conventions

### PR DESCRIPTION
**Pull request by WMDE-Fisch**
all test went well with the changes

### CODE REVIEW

**Comment by WMDE-Fisch** [comment]
for tldr; reviewers:

191 additions and 195 deletions.  <- 4 lines removed in .rubocop_todo.yml ;-)

**Comment by thiemowmde** [comment]
Needs rebase because of  #68.

File: tests/browser/features/step_definitions/entity_steps.rb [hunk 1]
```diff
@@ -6,41 +6,41 @@
 #
 # basic steps for entities
 
-Given /^I am logged in to the repo$/ do
+When(/^I am logged in to the repo$/) do
```
Comment (thiemowmde) [inline tests/browser/features/step_definitions/entity_steps.rb]: According to https://github.com/cucumber/cucumber/wiki/Given-When-Then you should not do this. I'm not sure how critical it is, probably technically irrelevant.

Comment (WMDE-Fisch) [inline tests/browser/features/step_definitions/entity_steps.rb]: Hmm true. First thought you could use "I am logged in to the repo" in a "Given" and a "When" context. But thinking about it the "When" context only makes sense when called "I log in to the repo". 

Will update the affected Lines :-)  - thanks for all the reviews btw...

e.g. I can use "Given I am logged in." to put the system in a known state, or I can use "When I am logged in."

**Comment by zeljkofilipin** [comment]
:+1:

**Pull request merged**